docs/overview.md

```diff
@@ -76,7 +115,8 @@ from my_app.models import User
 
 class ClickHouseUser(ClickHouseModel):
     django_model = User
-    
+    # Uncomment the line below if you want your models to sync between databases
+    # need_sync = True
```

Comment:
Where did your get this from? `need_sync()` is [a class method returning boolean](https://github.com/carrotquest/django-clickhouse/blob/master/src/django_clickhouse/clickhouse_models.py#L268), not attribute. If you want a model to be synced, you should set `sync_enabled` attribute, as it is metioned [here](https://github.com/carrotquest/django-clickhouse/blob/master/docs/synchronization.md#clickhousemodel-configuration)